casee.go

```diff
@@ -130,6 +130,35 @@ func IsFlatCase(s string) bool {
 	}
 }
 
+// Convert argument to UPPER_CASE style string.
+// If argument is empty, return itself.
+func ToUpperCase(s string) string {
+	if len(s) == 0 {
+		return s
+	}
+
+	fields := splitToLowerFields(s)
+	for i, f := range fields {
+		fields[i] = strings.ToUpper(f)
+	}
+	return strings.Join(fields, "_")
```

Comment:
I think that `strings.ToUpper(ToSnakeCase(s))` is simpler implementation.

Reply:
I can change implementation to this, I think it should work. Though I still think there should be method for UPPER_CASE even if it's so simply implemented - it's just convenient to have all methods in single module for casing conversions.